cognite/extractorutils/unstable/scheduling/_schedules.py

```diff
@@ -0,0 +1,34 @@
+from abc import ABC, abstractmethod
+from random import randint
+from time import time
+
+from croniter import croniter
+
+
+class Schedule(ABC):
+    @abstractmethod
+    def next(self) -> int:
+        pass
+
+
+class CronSchedule(Schedule):
+    def __init__(self, expression: str) -> None:
+        self._cron = croniter(expression)
+
+    def next(self) -> int:
+        return int(self._cron.get_next(start_time=time()))
+
+
+class IntervalSchedule(Schedule):
+    def __init__(self, interval: int) -> None:
+        self._interval = interval
+        self._jitter = randint(0, interval)
+
+        self._next = int(time() + self._jitter)
```

Comment:
I think jitter needs a rethink as discussed.

Reply:
Agree, I'll take it out for now and we can consider it later.